SimCalorimetry/EcalElectronicsEmulation/src/EcalSimRawData.cc

```diff
@@ -481,7 +481,7 @@ void EcalSimRawData::getSrfs(const edm::Event& event,
   //EE
   edm::Handle<EESrFlagCollection> hEeSrFlags;
   event.getByLabel(srDigiProducer_, eeSrFlagCollection_, hEeSrFlags);
-  for(size_t i=0; i < sizeof(eeSrf)/sizeof(int); ((int*)eeSrf)[i++] = -1){};
+  for(size_t i=0; i < (nEndcaps*nScX*nScY); ((int*)eeSrf)[i++] = -1){};
```

Comment:
I think this multiplication operation is going to perform for every iteration ..... may be better to calculate it once outside the loop?